news/2017-11-28-OWC-Round-of-16-recap.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu! World Cup 2017 Round of 16 Recap"
+date: 2017-11-28 14:00:00 +0000
+---
+
+16 teams took to the stage over the weekend aiming to make it through to next weekend, and in fact, they all did! While the Round of 16 is the only stage that doesn't see any teams getting knocked out, there is still plenty on the line. 
+
+![](/wiki/shared/news/banners/owc_2017_recap_banner.jpg)
+
+As we approach the end of November, the winter winds keep getting chillier. However, the Eliminations Stage that just began this past weekend only further fueled the BURNING LOVE of these players for their circle-clicking madness. Whether you were busy celebrating Thanksgiving with your family, or too involved in some shipgirl game’s event, here’s what you may have missed from the OWC 2017 Round of 16.
+
+The South Korean-Taiwanese rivalry has been among OWC’s longest-running ones. For many old players from the pre-2015 era, they can recall the stage that once belonged uniquely to [Dungeon](https://osu.ppy.sh/users/461720), [Sayonara-bye](https://osu.ppy.sh/users/713266), [Cookiezi](https://osu.ppy.sh/users/124493), [hvick225](https://osu.ppy.sh/users/50265), [Rucker](https://osu.ppy.sh/users/147515) and [Dabanlong](https://osu.ppy.sh/users/624254), to name a few. Over the years, South Korea has been the favoured team as they won against Taiwan in OWC 2013 Finals 6-5, OWC 2014 Semi-finals 6-5, and OWC 2016 Semi-finals 6-2. It almost seemed like fate has been toying with Taiwan. Prior to the match, Rucker recollected these past engagements on Twitter, voicing that Team Taiwan 2017 would like to end this streak of defeats.
+
+![](\wiki\shared\news\2017-11-28-OWC-Round-of-16-recap\Rucker_tweet_1.png)
+
+(A veteran’s only wish…)
+
+However, the Korean roster this year is certainly no joke, as we have not only the returning powerhouses like [firebat92](https://osu.ppy.sh/users/1777162) and [Yaong](https://osu.ppy.sh/users/1883865) but also new blood carries like [Pring](https://osu.ppy.sh/users/3478883) and [Pray](https://osu.ppy.sh/users/2190336). However, Team Taiwan, charging behind Rucker and [Flask](https://osu.ppy.sh/users/959763) – the old and new generation captains, managed to punish the Koreans severely every time a mistake was seen. The match came down to a 6-2 in favour of the Taiwanese fighters, and Rucker was more than happy to announce to the world that he could finally return a blow right back at South Korea for Taiwan.
```

Comment:
remove first ``However,`` replace with ``The Korean roster..``

new paragraph at However, Team Taiwan